crates/matrix-sdk-ui/src/timeline/polls.rs

```diff
@@ -37,17 +38,18 @@ pub(super) struct ResponseData {
 
 impl PollState {
     pub(super) fn new(content: NewUnstablePollStartEventContent) -> Self {
-        Self { start_event_content: content, response_data: vec![], end_event_timestamp: None }
+        Self { start_event_content: content, response_data: vec![], end_event_timestamp: None, has_been_edited: false }
     }
 
     pub(super) fn edit(
         &self,
         replacement: &NewUnstablePollStartEventContentWithoutRelation,
     ) -> Result<Self, ()> {
-        if self.response_data.is_empty() && self.end_event_timestamp.is_none() {
+        if self.end_event_timestamp.is_none() {
```

Comment:
When the edit starts on the client, there is no way to block incoming votes.
So we have decided to drop this check instead of raising an error.